pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -227,7 +227,8 @@ func (og *operationGenerator) GenerateVolumesAreAttachedFunc(
 			}
 		}
 
-		return nil, nil
+		// It is hard to differentiate migrated status for all volumes for verify_volumes_are_attached_per_node
```

Comment:
Do we actually do translation for the verify operations?

Reply:
We di translation when generate the VolumeSpec. So it does not matter if it is verify operation.

verify is a bulk operation thus is difficult to differentiate. It will check attachedVolumes(which is a collection of VolumeSpec), and these VolumeSpecs, it can be part CSI migrated spec and some real CSI spec. So we cannot tell the difference.